### DISCUSSION TITLE
Welche Wallbox kaufen?

### DISCUSSION THREAD

**Discussion by sirius-black-2** [camueller/SmartApplianceEnabler#353] [Q&A]
Hallo zusammen,

wie wahrscheinlich bei vielen hier, steht auch bei mir die Anschaffung eines E-Autos an. Gibt es eine Liste welche Wallboxen mit dem SAE gut funtionieren bzw. welche würdet ihr empfehlen?

Vielen Dank

**Comment by camueller** [comment]
Bei der Wahl der Wallbox solltest auch Du bedenken, dass Du einen speziellen FI-Schalter benötigst, der allein mit 250 Euro zu Buche schlägt: https://www.doepke.de/de/produkte/schuetzen/fehlerstromschutzschalter-rccb/fehlerstromtyp-a/fuer-die-elektromobilitaet/09134818

Außerdem benötigst Du einen 3-phasigen Zähler, der idealerweise per Modbus angebunden ist - ich nutze diesen: https://xn--stromzhler-v5a.eu/stromzaehler/drehstromzaehler/fuer-hutschiene-ungeeicht/262/sdm72dm-v2-3-phasen-zweirichtungs-drehstromzaehler-mit-rs485-und-s0

Als Wallbox nutze ich eine günstige mit dem verbreiteten Controller von PhoenixContact (EM-CP-PP-ETH - 2902802): 
https://esl-emobility.com/de/walli-light-elektroauto-ladestation-wallbox-mit-ladekabel-typ-2-22kw-32a-3-phasig.html
Aktuell steht da: Lieferung auf Anfrage.

Alternativ kannst Du auch einen go-eCharger verwenden. In diesem Fall benötigst Du keinen separaten FI-Schalter und auch keinen Zähler, da beides im go-eCharger enthalten ist.

Beide Wallboxen sind verbreitet und Du kannst sie auch mit anderen Smart-Home-Platformen nutzen, falls das irgendwann relevant sein sollte.

**Reply by sirius-black-2** [reply]
Vielen Dank für die Antwort!
Vielleicht können mir die Checker doch noch ein bisschen Support geben...
Bei mir ist es aktuell so, dass ein SMA Energymeter am Netzanschlußpunkt installiert ist und ein HM 1 und natürlich der SAE über den u.a. per S0 die Verbrauchsdaten der Wärmepumpe von einem separaten Zähle geloggt werden und meherere Verbraucher (Wasch-,Spülmaschine) über Sonoff pow (mit Tasmota) gesteuert werden, funkioniert soweit super!!
Jetzt kommt ein E-Auto und eine Wallbox (ein dreiphasiger 6 mm2 Anschluß im Carport und eine LAN Buchse sind vorhanden) dazu, evtl. würde ich den HM 1 (läuft schon über 8 Jahre) auch auf einen HM 2 "updaten".
Meine Frage: Der HM 2 kommt ja an die Stelle an der vorher der EM installiert war d.h. der wäre ja frei und könnte als Energiezähler für die Wallbox verwendet werden. Aus Bequemlichkeitsgründen würde ich evtl den SMA EV Charger installieren, in der Hoffnung die Einbindung ins Portal funktioniert dann gut. Allerdings weiß ich jetzt nicht, da der EV Charger ja per LAN mit dem Netzwerk (und den HM 2) verbunden ist, überhaupt einen separaten Zähler benötigt wird. Aus den Erklärvideos auf der SMA Seite werde ich leider nicht wirklich schlau...Vielleicht hat ja der ein oder andere eine ähnlich Konfiguration und kann mir seine Erfahrungen berichten. Da ich eh noch ca. 1 Jahr auf die Auslieferung warten muss, besteht auch kein Anlass zur Eile.
Vielen lieben Dank!!

**Reply by Trunks1982** [reply]
Ganz ehrlich spar dir das geld für den sma ev charger.😉

**Reply by camueller** [reply]
In Deinem Eingangspost hast Du gefragt, was zu empfehlen ist. Die Antwort has hast Du komplett ignoriert und schreibst jetzt, dasss Du "aus Bequemlichkeit den SHM EV Charger installieren" willst. Was soll man Dir da noch schreiben? Wenn Du unbedingt 1000 Euro mehr als sinnvoll ausgeben willst und eine Wallbox willst, bei der Du auf Gedeih und Verderb an SMA ausgeliefert bist, dann solltest Du das machen.

**Reply by sirius-black-2** [reply]
...natürlich habe ich mir Deine Empfehlungen angesehen!!! Der EV Charger schien mir einfach der einfachste Weg. Mein Ansinnen ist u.a. Althardware (EM) ggf. als Zähler weiterzuverwenden, wenn das mit dem Walli auch funktioniert, umso besser. Nicht falsch verstehen, als jemend der zwar einigermaßen mit einem PC umgehen kann, aber keinerlei Programierkenntnisse hat, habe ich einfach die Befürchtung, dass ich die Konfiguration nicht so ohne weiteres ans laufen kriege und hier wieder das Forum nerven muss...

**Reply by camueller** [reply]
Wenn Du wenig Stress willst und eine bewährte, verbreitete Wallbox, dann nimm den go-eCharger Gemini - kostet 759 und Du brauchst keinen separaten Zähler oder FI-Schalter. Es gibt hier diverse SAE-User mit dieser Wallbox.

https://shop.go-e.com/go-e-charger-gemini-22-kw

**Reply by sirius-black-2** [reply]
...kuck ich mir an, Danke!
